apps/web/components/booking/BookingListItem.tsx

```diff
@@ -298,7 +298,7 @@ function BookingListItem(booking: BookingItemProps) {
     const urlSearchParams = new URLSearchParams({
       allRemainingBookings: isTabRecurring.toString(),
     });
-    if (booking.attendees[0]) urlSearchParams.set("email", booking.attendees[0].email);
+    if (userEmail) urlSearchParams.set("email", userEmail);
```

Comment:
We can completely remove setting email in query params. AFAIK it is only used in the booking page to set rescheduledBy and cancelledBy fields, and since the current user already has a session, booking page can use the email from the session itself.